test/one_hot_encoding.jl

```diff
@@ -175,14 +175,14 @@
 
         df = DataFrame(:a => ["foo", "bar"], :b => ["foo2", "bar2"])
         expected = DataFrame(
-            [[1, 0], [0, 1], [0, 0], [0, 0], [0, 0], [0, 0], [0, 0], [0, 0], [1, 0], [0, 1]],
```

Comment:
These tests changed because what was happening before (in the Tables apply) when we had multiple columns is that we were calling _apply on each component in turn and then stacking the result. This lead to the original version that had many columns, which would have had duplicate names.

Now instead, we collect the components and call _apply on the collection, which is what we were doing for the Array method all along. So now we get one column per category (as expected) but the shape is now inconsistent.

So the previous result was inconsistent with the matrix application but I didn't notice that before. This version is now consistent. Either way I'm not too concerned about this because I'm not sure how much sense it makes to apply OHE to multiple columns. The output isn't necessarily valid or useful, so I would considering dropping the tests on multi-dimensional array altogether.